api/src/test/java/org/pmiops/workbench/utils/TestMockFactory.java

```diff
@@ -29,60 +31,84 @@
 import org.pmiops.workbench.model.WorkspaceAccessLevel;
 import org.pmiops.workbench.notebooks.model.ClusterStatus;
 import org.pmiops.workbench.notebooks.model.ListClusterResponse;
+import org.springframework.stereotype.Service;
 
+@Service
 public class TestMockFactory {
```

Comment:
I'm not really happy with the way this class works. It's not really freeing me from most of the burdens of setting up the test configuration, for example. And it's not really a factory.